host-bmc/host_pdr_handler.cpp

```diff
@@ -113,13 +113,14 @@ HostPDRHandler::HostPDRHandler(
     oemUtilsHandler(oemUtilsHandler)
 {
     isHostOff = false;
-    isHostTransitioningToOff = false;
+    isHostRunning = false;
```

Comment:
this is already i think added by @Pavithrab7 , i am not sure why isHostTransitioningToOff is being removed ?